### ISSUE TITLE
Just installed. Wired xbox360 controller not working.

### ISSUE THREAD

**Issue by supersaiyancurry** [recalbox/recalbox-os#500]
i installed 3.3.14 and plugged my xbox360 controller in and it's not working. The green light indication shows that I'm player 1, but I have no control and can't move out of the SNES part of the menu. I think this is an issue with the software because the controller is quite reliable.

**Comment by digitalLumberjack** [comment]
Is the cable reliable ?
Did you test on a computer, with the same cable ?

**Comment by supersaiyancurry** [comment]
Yes, the confroller is reliable. I've recently used it to play DOOM on my PC. I noticed something interesting though. After remaining inactive for an extended period of time- the screen would dim. When I pressed a button on my controller, the screen would brighten again- as if the controller did something. Also, I think this is unrelated, but I did run into a crash issue first time installing. I reinstalled though and I don't get any error messages now- just straight to the emulation station.

**Comment by digitalLumberjack** [comment]
And if you try to reconfigure the controller ? Plug a usb keyboard -> enter -> controllers -> configure

Use the SNES mapping for the buttons (https://github.com/recalbox/recalbox-os/wiki/Manual-%28EN%29#xboxcontrollers)

**Comment by supersaiyancurry** [comment]
Well, I don't have a USB keyboard on me right now. Sure, I'll try it out once I get my USB keyboard back from my parent's place, but it'd be nice if you could make it so the controller automatically works like the guide said it would because that's one of the primary reasons I downloaded this build: convenience.

**Comment by digitalLumberjack** [comment]
You can't blame us for having an exotic adaptor, as you can see on the issues and on the forum 99% of the xbox controllers work out of the box. We did configure the xbox 360 controllers, yours is just not the same.

**Comment by rockaddicted** [comment]
if your recalbox is connected to your network, and you have a smartphone, you can use virtualgamepads to navigate in recalbox menus.

https://github.com/recalbox/recalbox-os/wiki/Virtual-Gamepad-%28EN%29

**Comment by nadenislamarre** [comment]
can you retry in 4.1 ?
@rockaddicted some other bad feedback on it ?
